jupyter_server_proxy/_version.py

```diff
@@ -0,0 +1,4 @@
+# __version__ should be updated using tbump, based on configuration in
+# pyproject.toml, according to instructions in RELEASE.md.
+#
+__version__ = "4.1.1-0.dev"
```

Comment:
tbump is fine and all, but it could dispense with this copy with e.g.

```py
__version__ == __import__("importlib.metadata").metadata.version("jupyter_server_proxy")
```